metrics/helpers/lp.py

```diff
@@ -123,6 +123,52 @@ def get_team_daily_triage_count(team, distro, blacklist=None):
     return len(results)
 
 
+def get_team_unassigned_bugs(team, distro):
+    """Report count of new unassigned bugs for Launchpad team on a distro."""
+    lp_distro = LP.distributions[distro]
+    lp_team = LP.people[team]
+    return len(lp_distro.searchTasks(bug_subscriber=lp_team, assignee=None,
+                                     status='New'))
+
+
+def get_team_incomplete_bugs(team, distro):
+    """Report count of open bugs for Launchpad team on a distro."""
+    lp_distro = LP.distributions[distro]
+    lp_team = LP.people[team]
+    return len(lp_distro.searchTasks(bug_subscriber=lp_team,
+                                     status='Incomplete'))
+
+
+def get_mir_in_review():
+    """
+    Report count of MIRs in active review.
+
+    Open, Triaged or Confirmed (so not yet approved) bug would be assumed
+    to be "in active review".
+    """
+    lp_distro = LP.distributions['ubuntu']
+    lp_team = LP.people['ubuntu-mir']
+    return len(lp_distro.searchTasks(bug_subscriber=lp_team,
+                                     status=['Triaged', 'Confirmed']))
+
+
+def get_mir_in_security_review():
+    """Report count of open, assigned to Security bugs."""
+    lp_distro = LP.distributions['ubuntu']
+    lp_team = LP.people['ubuntu-mir']
+    assignee = LP.people['ubuntu-security']
+    return len(lp_distro.searchTasks(bug_subscriber=lp_team,
+                                     assignee=assignee))
```

Comment:
Does it matter if these bug tasks are Incomplete? Does the bug status change if the security team needs more information?

Reply:
Nah, I just consider this still up to security to finish their review.